packages/sdk-core/src/model/http/BacktraceRequestHandler.ts

```diff
@@ -14,12 +14,13 @@ export interface BacktraceRequestHandler {
         submissionUrl: string,
         dataJson: string,
         attachments: BacktraceAttachment[],
+        abort?: AbortSignal,
     ): Promise<BacktraceReportSubmissionResult<BacktraceSubmissionResponse>>;
 
     /**
      * Post data to Backtrace API
      * @param submissionUrl data submission URL
      * @param payload request payload
```

Comment:
we need to add abort signal definition to the function description

Reply:
Done.